src/ReceivingList/ReceivingList.js

```diff
@@ -35,7 +35,7 @@ const visibleColumns = ['title', 'poLine.receiptDate', 'poLine.titleOrPackage',
 const sortableFields = ['title', 'poLine.receiptDate', 'poLine.titleOrPackage', 'poLine.poLineNumber', 'poLine.receivingNote'];
 const columnMapping = {
   'title': <FormattedMessage id="ui-receiving.titles.title" />,
-  'poLine.receiptDate': <FormattedMessage id="ui-receiving.title.expectedDate" />,
+  'poLine.receiptDate': <FormattedMessage id="ui-receiving.title.expectedReceiptDate" />,
```

Comment:
my ticket says it should be `Expected date`

Reply:
But mine has more fresh comment from Dennis, I believe this name should be consistent